dpb/settings.py

```diff
@@ -137,30 +137,35 @@
 # Filer Settings
 FILER_CANONICAL_URL = 'filer/'
 
-# Markdown Deux Settings
-MARKDOWN_DEUX_STYLES = {
-    "default": {
-        "extras": {
-            "code-friendly": None,
-        },
-        # Allow raw HTML (WARNING: don't use this for user-generated
-        # Markdown for your site!).
-        "safe_mode": False,
-    },
-    "nohtml": {
-        "extras": {
-            "code-friendly": None,
-        },
-        # Allow raw HTML (WARNING: don't use this for user-generated
-        # Markdown for your site!).
-        "safe_mode": "escape",
-    },
-}
-# END MARKDOWN DEUX
-
 # Configure Easy Thumbnails
 THUMBNAIL_ALIASES = {
     '': {
         'pages': {'size': (350, 350), 'crop': "scale", 'quality': 100},
     },
 }
+
+MARKDOWNIFY_WHITELIST_TAGS = [
+    'table',
+    'thead',
+    'tbody',
+    'th',
+    'tr',
+    'td',
+    'a',
+    'abbr',
+    'acronym',
+    'b',
+    'blockquote',
+    'em',
+    'i',
+    'li',
+    'ol',
+    'p',
+    'strong',
+    'ul',
+    'script',
```

Comment:
Gibt es einen Grund, dafür, dass wir `script` Tags brauchen? 

Werden Markdown Ressourcen nur von uns erstellt?

Reply:
Der commit dazu hieß `Add script for more fun` :D

Markdown ist unglaublich dumm. Kp warum sich das so krass durchgesetzt hat. Man kann keine Klassen hinzufügen zu Entitäten. Aber mit JS geht das

```
| Titel        | Quelle           | Audio  |
| ------------- |-------------| -----|
| Abends treten Elche |  | [mp3](irgendeinlink) |
| col 2 is      | centered      |   $12 |
| zebra stripes | are neat      |    $1 |







<!-- Diese Zeilen machen die Tabelle etwas hübscher. Einfach stehen lassen :-) -->
<script>
var tables = document.getElementsByTagName("table");
for (var table of tables) {
    table.classList.add("table", "table-striped", "table-hover")
}
</script>
```